ua-snap/iem-webapp
# Adds protected areas to app

Testing/checking:

 - In the Place selector, observe that the descriptive text above the box notes that protected areas can be searched.
 - Place selector: search for Chugach.  Observe that there's lots of protected areas matching that.
 - Place selector: search for USFS.  There's matches (this confirms that the type of protected area is searched along with its name, so you can search for NPS, etc).
 - Choose the Chugach Backcountry Prescription and trigger the search.  Results should show, and the place name should show up throughout but _without_ the long description of the type of protected area (in this case, "Backcountry Prescription (USFS)").
